src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -547,7 +547,7 @@ private static OutputStreamOptions getOutputStreamOptions(Path outputPath) {
      * @noinspection PackageVisibleInnerClass
      * @noinspectionreason PackageVisibleInnerClass - we keep this enum package visible for tests
      */
-    enum OutputFormat {
+    /* package */ enum OutputFormat {
         /** XML output format. */
         XML,
```

Comment:
lets put this to new issue to review, kind of strange that not making it public, we usually do not use package-private just for tests.

Reply:
https://github.com/checkstyle/checkstyle/issues/14870